webapp-django/crashstats/signature/static/signature/js/signature_tab_reports.js

```diff
@@ -97,11 +103,13 @@ SignatureReport.ReportsTab.prototype.getParamsForUrl = function () {
         });
     }
 
+    // Get the sort for the input.
+    params._sort = this.$sortInputHidden.val().trim().split(',') || [];
```

Comment:
That `|| []` is never going to be needed. `['']` or `['one','two']` is always true.